Tools/Scripts/libraries/webkitcorepy/setup.py

```diff
@@ -30,7 +30,7 @@ def readme():
 
 setup(
     name='webkitcorepy',
-    version='0.12.1',
+    version='0.12.2',
```

Comment:
Is there a way to sync this version with the version in `__init__.py`?

Reply:
Yes, but it's not trivial. I have a partial change to do it, but it's not particularly high-priority at the moment.